packages/legend-studio/src/models/protocols/pure/PureProtocolProcessorPlugin.ts

```diff
@@ -60,5 +70,5 @@ export abstract class PureProtocolProcessorPlugin extends AbstractPlugin {
    * sending the server additional elements not needed for execution. This would provide a mechanism
    * to add more elements in this reduced graph.
    */
-  V1_getExtraExecutionInputElements?(): V1_PackageableElement[];
+  V1_getExtraExecutionInputSelectors?(): V1_ExecutionInputSelector[];
```

Comment:
Selector or Getter? I would say Getter because you can potentially change the element as well if you wish to do so